<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatClient</class>
 <widget class="QWidget" name="ChatClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="logGroupBox">
      <property name="title">
       <string>Log In</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_pic">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QFormLayout" name="logformLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="nameLabel">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="ipAddressLabel">
               <property name="text">
                <string>IP Address</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="ipAddressLineEdit"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="portNumLabel">
               <property name="text">
                <string>Port No</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="portNumLineEdit"/>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="nameLineEdit"/>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="logPushButtonLayout">
             <item>
              <widget class="QPushButton" name="logInPushButton">
               <property name="text">
                <string>Log In</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="logOutPushButton">
               <property name="text">
                <string>Log Out</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string>IP</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Port</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Message</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="chattingGroupBox">
      <property name="title">
       <string>Chatting</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="chatInPushButton">
           <property name="text">
            <string>Chat In</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="messageTextEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="messageLineEdit"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="chattingPushButtonLayour">
           <item>
            <widget class="QPushButton" name="fileTransferPushButton">
             <property name="text">
              <string>File Transfer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="sendPushButton">
             <property name="text">
              <string>Send</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
